torch/csrc/generic/StorageMethods.cpp

```diff
@@ -317,24 +338,43 @@ PyObject * THPStorage_(_setCdata)(THPStorage *self, PyObject *new_cdata)
 }
 
 static PyMethodDef THPStorage_(methods)[] = {
-  {"copy_", (PyCFunction)(void(*)(void))THPStorage_(copy_), METH_VARARGS | METH_KEYWORDS, nullptr},
-  {"element_size", (PyCFunction)THPStorage_(elementSize), METH_NOARGS, nullptr},
-  {"fill_", (PyCFunction)THPStorage_(fill_), METH_O, nullptr},
-  {"new", (PyCFunction)THPStorage_(new), METH_NOARGS, nullptr},
-  {"resize_", (PyCFunction)THPStorage_(resize_), METH_O, nullptr},
-  {"size", (PyCFunction)THPStorage_(size), METH_NOARGS, nullptr},
-  {"data_ptr", (PyCFunction)THPStorage_(dataPtr), METH_NOARGS, nullptr},
-  {"is_pinned", (PyCFunction)THPStorage_(isPinned), METH_NOARGS, nullptr},
-  {"_write_file", (PyCFunction)THPStorage_(writeFile), METH_VARARGS, nullptr},
-  {"_new_with_file", (PyCFunction)(void(*)(void))THPStorage_(newWithFile), METH_O | METH_STATIC, nullptr},
-  {"_set_from_file", (PyCFunction)THPStorage_(setFromFile), METH_VARARGS, nullptr},
+    {"copy_",
+     (PyCFunction)(void (*)(void))THPStorage_(copy_),
+     METH_VARARGS | METH_KEYWORDS,
+     nullptr},
+    {"element_size",
+     (PyCFunction)THPStorage_(elementSize),
+     METH_NOARGS,
+     nullptr},
+    {"fill_", (PyCFunction)THPStorage_(fill_), METH_O, nullptr},
+    {"new", (PyCFunction)THPStorage_(new), METH_NOARGS, nullptr},
+    {"resize_", (PyCFunction)THPStorage_(resize_), METH_O, nullptr},
+    {"size", (PyCFunction)THPStorage_(size), METH_NOARGS, nullptr},
+    {"set_nbytes_", (PyCFunction)THPStorage_(setNbytes_), METH_O, nullptr},
```

Comment:
@ezyang , I added `set_nbytes_()` and `nbytes()` to the python api. Do you think that's a good idea, or would it be better not to?

Reply:
Better not, unless there's a reason you needed it